rclpy/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ function(configure_python_c_extension_library _library_name)
   )
 
   install(TARGETS ${_library_name}
-    DESTINATION "${PYTHON_INSTALL_DIR}/${PROJECT_NAME}"
+    DESTINATION "${PYTHON_INSTALL_DIR}/${PROJECT_NAME}_impl"
```

Comment:
This package doesn't own the namespace `rclpy_impl`. If another package with that name installs its files they will collide with this one.